### PR TITLE
Doc asistent llistes preus

### PR DESCRIPTION
**Pull request by eberloso**
### Description:

Documentar com s'utilitza l'asistent per carregar llistes de preus.

### Tasks:

- [X] Documentacio
- [x] Traduccions

### Links:

* [Llistes de preus](http://builds.gisce.net/powerp-docs/powerp_doc_llistes_preu/ca/comer/facturacion/#llistes-de-preus)
* [Listas de precios](http://builds.gisce.net/powerp-docs/powerp_doc_llistes_preu/es/comer/facturacion/#llistes-de-preus)

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_doc_llistes_preu/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_doc_llistes_preu/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_doc_llistes_preu/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_doc_llistes_preu/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/comer/facturacion.md [hunk 1]
```diff
@@ -4,6 +4,50 @@
 
 Les llistes de preus....
 
+### Asistent per carregar llistes de preus
+
+Mitjançant l'assistent "Create pricelist from file" situat al menú **Productes -> Tarifes -> Create pricelist from file**
```
Comment (jaumef) [inline docs/comer/facturacion.md]:
```suggestion
Mitjançant l'assistent "`Create pricelist from file`" situat al menú **Productes -> Tarifes -> Create pricelist from file**
```

File: docs/comer/facturacion.md [hunk 2]
```diff
@@ -4,6 +4,50 @@
...
+- **Fitxer de la tarifa**: fitcher csv a carregar. Conté una linia per cada tarifa d'acces a la qual se li volen definir els preus en la versio de la llista de preus.
+- **Data inici**: data inici de la versió de la llista de preus a actualitzar/crear.
+- **Data final**: data final de la versió de la llista de preus a actualitzar/crear. Si es deixa buit, la versió creada no tindrà data final.
+
+
+#### Format del fitxer CSV
+El fitxer ha de ser en format CSV **separat per ";"** i ha de tenir les següents columnes:
+
+- **Tarifa d'Accés**: nom de la tarifa d'accés per la qual es crearan les regles que en definiran els preus. Es crearà una regla per cada periode d'aquesta tarifa.
+- **Llista de preus**: nom de la llista de preus a crear/actualitzar. El ERP buscarà una llista de preus amb aquest nom, si no la troba la crearà. Després buscarà una versió d'aquesta llista que comenci en la data inici introduïda en l'assistent. Si no la troba també la crearà. Totes les regles creades per la tarifa d'accés indicada s'afegiran en aquesta versió. A més, es farà que la tarifa d'acces sigui compatible amb aquesta llista de preus.
```
Comment (jaumef) [inline docs/comer/facturacion.md]: No está traducido al `es_ES`, revisa las traducciones del fichero PO

File: locales/es_ES/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -9666,6 +9666,168 @@ msgstr "Lista de precios"
...
+
+msgid ""
+"<strong>Llista de preus</strong>: nom de la llista de preus a crear/actualitzar. El "
+"ERP buscarà una llista de preus amb aquest nom, si no la troba la crearà. Després "
+"buscarà una versió d'aquesta llista que comenci en la data inici introduïda en "
+"l'assistent. Si no la troba també la crearà. Totes les regles creades per la tarifa "
+"d'accés indicada s'afegiran en aquesta versió. A més, es farà que la tarifa d'acces "
+"sigui compatible amb aquesta llista de preus."
+msgstr ""
+"<strong>Llista de preus</strong>: nombre de la lista de precios a crear/actualizar. "
```
Comment (jaumef) [inline locales/es_ES/LC_MESSAGES/messages.po]: Missing translation

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_doc_llistes_preu/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_doc_llistes_preu/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_doc_llistes_preu/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Pull request merged**